code/modules/antagonists/cult/ritual.dm

```diff
@@ -40,17 +40,22 @@ This file contains the cult dagger and rune list code
 	scribe_rune(user)
 
 /obj/item/melee/cultblade/dagger/proc/scribe_rune(mob/living/user)
```

Comment:
There's a call to this below when input fails, does that work correctly with this ?